### ISSUE TITLE
(Javascript) Float32Array broken after 11.0.1 

### ISSUE THREAD

**Issue by fserb** [highlightjs/highlight.js#3252]
**Describe the issue**

`Float32Array` is being broken in 3 token `Float` `32` and `Array`. This used to work fine on 10.7.3. Here is a [side-by-side result jsfiddle](http://jsfiddle.net/xbzwndrp/5/).

**Which language seems to have the issue?**

javascript

**Are you using `highlight` or `highlightAuto`?**

`highlightElement`

**Sample Code to Reproduce**

```
new Float32Array();
```

also repros with more complicated lines, like `const x = new Float32Array(3);`.

**Expected behavior**

What used to happen on 10.7.3. `Float32Array` highlighted as a builtin.

**Additional context**

**Comment by joshgoebel** [comment]
This is indeed broken currently, although the new behavior may be to highlight this simply as a `title.class` rather than `built-in`.  The  CLASS_REFERENCE mode likely just needs to be expanded to proper recognize numbers and underscores in addition to ASCII.

**Comment by fserb** [comment]
Makes sense. I'm fine with either way as long as it recognizes it as a single entity. :)

**Comment by rbrishabh** [comment]
I can help with this!

**Comment by joshgoebel** [comment]
PR welcome.
